codespell_lib/_codespell.py

```diff
@@ -50,6 +50,10 @@
 )
 _builtin_default = 'clear,rare'
 
+# docs say os.EX_USAGE is only available on Unix systems, and it is 64 on
+# macOS, but things like bash and flake8 give 2, so let's just KISS and use:
+EX_USAGE = 2
+
```

Comment:
If you're going to call it `EX_USAGE`, why not use the same value (`64`) as defined by BSD 30 years ago?

Reply:
Agreed, I errantly thought it was `2` on Unix and `64` on macOS -- I've now changed it to use `EX_USAGE`/`64`

Reply:
I'd just do `EX_USAGE = 64` and forget about `os.EX_USAGE`. Python made a big mistake in reading this from the header instead of just hard-coding it. The way they define it in the code and the documentation naturally leads people to believe the value varies by platform, but it doesn't -- it's either `64` or the header doesn't exist. Having the the value be `64` everywhere is the whole point of semantic exit codes. Then you can just say "our exit codes are documented in `man sysexits`".

Reply:
I quite like the import option, we do similar in something else I work on:
https://github.com/OpenLightingProject/ola/blob/master/common/base/SysExits.cpp